_docs/1.0.0/usage-schema.md

```diff
@@ -0,0 +1,618 @@
+| `%`    | Prefix or suffix      | Divide by 100 on parsing                                                                                                                                            |                                                                                    |
+| `‰`    | Prefix or suffix      | Divide by 1000 on parsing                                                                                                                                           |                                                                                    |
+| `'`    | Prefix or suffix      | Used to quote special characters in a prefix or suffix, for example, "'#'#" formats 123 to "#123". To create a single quote itself, use two in a row: "# o''clock". |                                                                                    |
+| `∞`    | *not part of pattern* | String ti represent infinity                                                                                                                                        |                                                                                    |
+
+<a name="pattern-parsing-star"/>\* while these can ne changed, in the [`pattern`]($pattern) the default value *must* be 
+used. E.g. *"."*, *"."* and *"-"*.
+
+**NB!** If there's no special format of the input, it's advised to avoid the usage of patterns due to the added
+performance hit (parsing using a pattern adds computational overhead). It's still possible to redefine the special
+symbols like [decimal separator](#decimal_separator) or [minus sign](#minus_sign) even without the pattern.
+
+#### Number parsing peculiarities
+
+- number of `#` and `0` in pattern is not reliable, stuff is parsed regardless (e.g. *"0.0"* produces the same result as
+*"##.#*)
+- Grouping signs placement "flexible"; in other words if grouping separator has been added to the pattern the number is
+parser regardless of the position (e.g. pattern `#,##0.#` will parse _1,000.0_, _1,1234.5_ so as  _2000._)
+- Grouping sign not accepted in decimal places
+- _"+"_ not accepted if pattern is specified unless explicitly mentioned
+- `%` and `‰` in fractional and decimal values and patterns divide the number by 100 and 1000 (in integrals as well, but
+then they are usually not whole numbers)
+- Even if redefined  the standard grouping and decimal separators **needs to be used** in the pattern
+- If pattern is used, `e` is not accepted only `E` in the exponential expresion (without a pattern both are recognized)
+
+Defaults
+--------
+
+As it was described, in case when the standardization is unsuccessful, either due to missing data in a column
+non-nullable column or because the casting fails, default value is used in that particular column and row.
```

Comment:
```suggestion
or because it was being cast to the wrong type, the filed is populated with a default value and an error is added to the error column.
```